config.toml

```diff
@@ -0,0 +1,5 @@
+# configuration for automated builds
```

Comment:
This is good to separate out from the YAML, per above. ☝🏻 

This configuration requires manual updates during every release cycle. If we add the update to the release cycle playbook, does that also mean configuring repo permissions for the release lead and their team to make and approve changes?

Reply:
Ideally, I'd like to use the configuration from `website` and not maintain a separate copy here.
The "action" would read (or pull in) config.toml.